action.yml

```diff
@@ -1,8 +1,8 @@
 name: "Update docs"
-description: "Generate markdown documentation, commit it to a branch of the gitbook repo, and push it"
+description: "Generate markdown documentation, commit it to a branch of the docodile repo, and create pull request"
```

Comment:
I think this description is out of date now that we're back to commit and push

Reply:
Good eye. Updated in latest commit.